rsocket-core/src/main/java/io/rsocket/ConnectionSetupPayload.java

```diff
@@ -16,117 +16,115 @@
 
 package io.rsocket;
 
-import static io.rsocket.frame.FrameHeaderFlyweight.FLAGS_M;
-
 import io.netty.buffer.ByteBuf;
 import io.netty.util.AbstractReferenceCounted;
-import io.rsocket.Frame.Setup;
+import io.rsocket.frame.FrameHeaderFlyweight;
 import io.rsocket.frame.SetupFrameFlyweight;
-import io.rsocket.framing.FrameType;
 
 /**
  * Exposed to server for determination of ResponderRSocket based on mime types and SETUP metadata/data
  */
 public abstract class ConnectionSetupPayload extends AbstractReferenceCounted implements Payload {
-
-  public static ConnectionSetupPayload create(final Frame setupFrame) {
-    Frame.ensureFrameType(FrameType.SETUP, setupFrame);
+  
+  public static ConnectionSetupPayload create(final ByteBuf setupFrame) {
     return new DefaultConnectionSetupPayload(setupFrame);
   }
-
+  
```

Comment:
Are we having some format war? Every PR recently seems to reformat more lines than it changes...

Reply:
I didn't run a formatter on purpose I was going to wait till after this pull request and then make the formatter required for a build to pass.